.env.example

```
@@ -0,0 +1,111 @@
+# Production Environment Variables - متغيرات بيئة الإنتاج
+# Real production configuration - إعدادات إنتاج حقيقية
+
+# Application Settings - إعدادات التطبيق
+APP_NAME=Super Blue Badge App
+APP_VERSION=1.0.0
+APP_ENV=production
+APP_DEBUG=false
+APP_SECRET_KEY=your-super-secret-key-here-change-in-production
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The APP_SECRET_KEY key should go before the APP_VERSION key

(UnorderedKey)